Doc/library/argparse.rst

```diff
@@ -185,6 +240,8 @@ ArgumentParser objects
 The following sections describe how each of these are used.
 
 
+.. _prog:
+
 prog
 ^^^^
```

Comment:
I think the target markup is not necessary.  The headings are automatically targets, but the markup to create the link is different (if I remember right):

```
`:ref:`prog`
```